ansible_galaxy/build.py

```diff
@@ -16,7 +16,7 @@
 
 log = logging.getLogger(__name__)
 
-ARCHIVE_FILENAME_TEMPLATE = 'v{version}.{extension}'
+ARCHIVE_FILENAME_TEMPLATE = '{name}-{version}.{extension}'
 ARCHIVE_FILENAME_EXTENSION = 'tar.gz'
```

Comment:
Unrelated to this pr, but this reminds me that we can probably change the default format to 'tar.bz2' for slightly smaller artifacts.

Reply:
I think `gzip` is de-facto a standard. As an alternative we might want consider `.zip` format  for better portability.